src/components/Main/index.jsx

```diff
@@ -25,12 +24,15 @@ const selectorGenerator = new CssSelectorGenerator({
   selectors: ['id', 'tag', 'nthchild'],
 });
 
-const getWrappedText = (node) => {
-  const textNodes = filter(
-    node.childNodes, child => child.nodeType === child.TEXT_NODE);
-  const combined = trim(
-    textNodes.map(textNode => textNode.textContent).join(''));
-  return combined || null;
+const getWrappedText = (node, rich = false) => {
+  if (node.nodeType === node.TEXT_NODE) {
+    return node.textContent;
+  }
+  if (node.tagName === 'BR') {
```

Comment:
I think that's harder since the div could be used to style content without actually spacing it out, but I don't feel strongly